releases/v21_0_0.rst

```diff
@@ -130,9 +130,9 @@ For more information, refer to :jira:`RFC-736`.
 .. _release-v21-0-0-pipelines_check:
 
 Addition of the pipelines_check package
------------------------------------
+---------------------------------------
 
-The new :ref:`pipelines_check <lsst.pipelines_check>` package replaces `lsst_dm_stack_demo`, which relied on the now :ref:`release-v21-0-0-remove-obs_sdss<removed obs_sdss>`. This new package demonstrates the processing of a single CCD with the Science Pipelines. It contains input files and astrometric reference catalogs needed to process the data from a single HSC detector. Its main purpose is to check that the pipeline infrastructure is working correctly.
+The new :ref:`pipelines_check <lsst.pipelines.check>` package replaces `lsst_dm_stack_demo`, which relied on the now :ref:`removed obs_sdss<release-v21-0-0-remove-obs_sdss>`. This new package demonstrates the processing of a single CCD with the Science Pipelines. It contains input files and astrometric reference catalogs needed to process the data from a single HSC detector. Its main purpose is to check that the pipeline infrastructure is working correctly.
```

Comment:
There is no `lsst.pipelines.check` so why do we need this change? pipelines_check isn't a python package.

Reply:
If there's no target, I'll just remove the reference altogether.